### PR TITLE
Imported latest codes + added ULCB and ULBE

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* None

The reason for this change is to restore a couple of conversion records that were deleted (and also use the latest NR codes enum).

*Description of changes:*
- app version = 5.0.9
- imported latest shared enums
- added conversion types ULCB and ULBE from main branch
- fixed conversion desc and text capitalizations

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by JazzarKarim — APPROVED** [review]

File: src/list-data/conversion-types.ts [hunk 1]
```diff
@@ -43,5 +43,25 @@ export const ConversionTypes: ConversionTypesI[] = [
...
     shortlist: false
+  },
+  {
+    desc: 'Unlimited Liability Company to a limited Company',
+    text: 'Unlimited Liability Company to a limited Company',
+    entity_type_cd: EntityType.CR,
+    blurbs: [
+      'Alter business type from an unlimited Liability Company to a limited Company.'
+    ],
+    value: NrRequestTypeCodes.CONVERT_ULCB,
```
Comment (severinbeauvais) [inline src/list-data/conversion-types.ts:54]: See all known NR Request Type codes here:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/name-request-codes.ts

Note that we don't support all of them yet, which was the source of Mihai's concern this morning. He's looking at the new codes in that table and will create tickets for missing functionality.

Comment (severinbeauvais) [reply src/list-data/conversion-types.ts:54]: Background/aside: some of those new codes are used for "Societies" (which doesn't use Namerequest UI or LEAR yet, but may be added soon).

File: src/list-data/conversion-types.ts [hunk 2]
```diff
@@ -43,5 +43,25 @@ export const ConversionTypes: ConversionTypesI[] = [
...
+  },
+  {
+    desc: 'Unlimited Liability Company to a Benefit Company',
+    text: 'Unlimited Liability Company to a Benefit Company',
+    entity_type_cd: EntityType.BC,
+    blurbs: [
+      'Alter business type from an ULC to a Benefit Company.'
+    ],
+    value: NrRequestTypeCodes.CONVERT_ULBE,
+    shortlist: false
```
Comment (severinbeauvais) [inline src/list-data/conversion-types.ts:65]: By the way, does anyone know why these 2 records were deleted? Mihai/Janis think they should be there, which is why I restored them.

Comment (JazzarKarim) [reply src/list-data/conversion-types.ts:65]: I was wondering why were they deleted in the first place. No idea Sev.

**Review by chenhongjing — APPROVED** [review]

File: src/list-data/conversion-types.ts (hunk 2 above)
Comment (severinbeauvais) [reply src/list-data/conversion-types.ts:65]: Ah, this is why: https://github.com/bcgov/namerequest/pull/621

Comment (JazzarKarim) [reply src/list-data/conversion-types.ts:65]: Oh, this is the commit that the branch is behind on in regards to main. OK, makes sense.

**Pull request merged**